### ISSUE TITLE
What does the release process look like?

### ISSUE THREAD

**Issue by CoreyKovalik** [aerogear/graphback#2299]
> If you have a question about Graphback that is not a bug report or feature
> request, feel free to post it here.
> 
> You can also reach the Graphback team on [Discord](https://discord.gg/mJ7j84m).
> 

What does the release process look like for this project? I see that the last release as of today is v1.1.2 released on Feb 24, 2021.

If I were to commit support for case sensitivity (see https://github.com/aerogear/graphback/issues/1967) for Knex, it would seem like it would be very slow moving to actually release into production.

It seems that there's a few options.

1. Push a PR to graphback
2. Fork and maintain our own graphback.
3. Patch locally, and do some messy code to extend our data provider & service creator / copy & pasting code to support **just to add** `ilike` to the knex query mapper.

Option 1 is ideal, but it seems that it may never make its way into production.
Option 2 seems best due to Option 1's downside.
Option 3 is soooo far from desirable

**Comment by wtrocki** [comment]
Hi

Apologies for lack of release. Our team moved out of the project temporarily and making fewer changes. 
However, we going to reply and keep things alive for a while. If you would like to fork it or even become a fully qualified maintainer of the project let us know.
For the moment we can provide latest release although there is no feature development happening on the project for the next 3-4 months timeframe

**Comment by wtrocki** [comment]
Apologies for the lack of release. Our team moved out of the project temporarily and making fewer changes. 
However, we going to reply and keep things alive for a while. If you would like to fork it or even become a maintainer of the project let us know
For the moment we can provide the latest release although there is no feature development happening on the project for the next 3-4 months timeframe.

**Comment by wtrocki** [comment]
TL;DR - you can become maintainer, get permission to release and work with Option 1 indefinitely or fork it and replicate entire infrastructure.

**Comment by CoreyKovalik** [comment]
@wtrocki Thank you for the transparency! I would be interested in becoming a maintainer in the near future.  We're soon implementing more searching/filtering features and finding that the #1967 issue needs some attention :)

**Comment by wtrocki** [comment]
Release process involves creating tag with the new version in the releases. CI/CD does the rest (unless templates were changed) then we need to push main https://github.com/aerogear/graphback/tree/templates-1.0.0 this branch to release templates.

Hope that answers question. Closing for now
